src/main/java/com/mfarag/todo/TaskList.java

```diff
@@ -1,8 +1,31 @@
 package com.mfarag.todo;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class TaskList {
 
-    public Object getTask() {
-        return null;
+    List<String> list = new ArrayList<>();
+
+
+    public String getTask(int id) {
+            if (id >= list.size()) {
+                return null;
+            } else {
+                return list.get(id);
+
+            }
+    }
+
+    public String getFirstTask () {
+        if (list.size() > 0) {
```

Comment:
The code inside this method is duplicate to the code inside `getTask`, you could have instead called `getTask(0)` inside this method.

That said, we don't need two methods. We only need one and you should feel free to update other tests accordingly.

Reply:
Yes, I was going to do that, but I thought that you want me to write a separate method for every test..